app/models/user.rb

```diff
@@ -9,9 +9,13 @@ class User < ApplicationRecord
 
   delegate :account_number, to: :account
 
-  def authorized_to?(access_request)
-    rbac_permissions.any? do |access|
-      Rbac.verify(access.permission, access_request)
+  def authorized_to?(*access_requests)
+    return true if ActiveModel::Type::Boolean.new.cast(Settings.disable_rbac)
+
+    access_requests.flatten.all? do |access_request|
```

Comment:
Same as wrote [previously](https://github.com/RedHatInsights/compliance-backend/pull/1261/files#r861482515), you'd need to do `any?` in order to allow both the view and the reporting.